Cargo.toml

```diff
@@ -40,12 +40,13 @@ derivative = "1.0.3"
 version = "3.0.0"
 
 [dev-dependencies]
-parsec-client-test = { git = "https://github.com/parallaxsecond/parsec-client-test", tag = "0.3.0" }
 num_cpus = "1.10.1"
 picky-asn1-der = "0.2.2"
 picky-asn1 = "0.2.1"
 serde = { version = "1.0", features = ["derive"] }
 sha2 = "0.8.1"
+parsec-client = { git = "https://github.com/parallaxsecond/parsec-client-rust" }
```

Comment:
Let's publish it as a crate! I think it would make sense to do it now and pull it as a normal crates.io dependence! To get rid of the different versions in dependences issue.

Reply:
I'm sorta relying on the CI to tell me if the client actually works 😅  if it does, will replace!

Reply:
I see, makes sense!